### PR TITLE
OP-1008 Clarify New Attendance and Re-Attendance fields

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-1008

### CODE REVIEW

**Review by mwithi — CHANGES_REQUESTED** [review]
With the aim of better clarifyinf the two fields, I would put in the "OPD Extended" chapter, a reference to the **Type of attendance** above or... to write again a text including the "Re-Attendance" behavior

**Comment by dbmalkovsky** [comment]
I've attempted to add additional information for the fields.

File: doc_user/UserManual.adoc [hunk 1]
```diff
@@ -260,11 +260,33 @@ The OPD Extended module affords the following additional features:
...
-* Once the patient is selected and a previous OPD visit is found, special fields are populated with the most recent visit of the patient (4), his/her personal data (5), and the Re-Attendance checkbox will be selected automatically upon selection of the main (first) disease (6);
+* radio box (5) to distinguish whether an individual is new (*New Attendance*)
+ or a (*Re-Attendance*) in the ambulatory. The *New Attendance*
+ selection indicates that the patient came to the ambulatory to report a
+ new health issue or new disease for which no care has been previously
+ provided for and/or addressed. This could be their initial visit or their n-th
+ time in the ambulatory, it does not matter in regard to this selection.
+ The important criteria for selecting *New Attendance* is that the problem
+ is under consideration is a new issue; that is, one that has not been dealt
```
Comment (mwithi) [inline doc_user/UserManual.adoc]:
```suggestion
 under consideration is a new issue; that is, one that has not been dealt
```

File: doc_user/UserManual.adoc [hunk 2]
```diff
@@ -260,11 +260,33 @@ The OPD Extended module affords the following additional features:
...
+ with previously. For
+ an issue or disease previously dealt with but not completely addressed (that
```
Comment (mwithi) [inline doc_user/UserManual.adoc]:
```suggestion
 with previously. For an issue or disease previously dealt with but not completely addressed (that
```

**Review by mwithi — APPROVED** [review]

**Pull request merged**